### ISSUE TITLE
Have a Galasa_Home variable to mark location of files etc

### ISSUE THREAD

**Issue by ChrisAtkinson1** [galasa-dev/projectmanagement#1397]
It would be perhaps useful to have an Environment Variable eg Galasa_Home

This could then be used to set where local run outputs could be stored ( eg Galasa_Home\localrun )

Also the properties could be stored here ( eg Galasa_Home\properties\ )

This would negate having to put stuff under \Users\user\blah.  Which on Windows is sometimes security restricted.

**Comment by techcobweb** [comment]
Yes we could do this, and if it's not set, then fall back to the previous setting.

Also, we could add this as an explicit command-line parameter to the CLI commands which touch this location.

Would need changing in eclipse also, so that eclipse and the cli tools respected the same location. So we could use a system environment variable in Windows, but probably also need a settings value in the eclipse panels to allow it to be set also.

**Comment by techcobweb** [comment]
I think the eclipse part could be split off into a separate story to be implemented separately at a different time. Thoughts ?

**Comment by ChrisAtkinson1** [comment]
As long as the split is CLI v all IDE's not just eclipse yes.
